MerlinAU.sh

```diff
@@ -2308,7 +2362,7 @@ Please manually update to version $minimum_supported_version or higher to use th
         fi
     fi
 
-    availableRAM_kb=$(_GetAvailableRAM_KB_)
+    availableRAM_kb=$(get_free_ram)
```

Comment:
This is incorrect. We want to know the amount of "**available**" RAM, **not** the amount of "**free**' RAM, for the reasons explained in my previous comment for PR #33.